amep/evaluate.py

```diff
@@ -3678,8 +3682,9 @@ def __init__(
             Skip this fraction at the beginning of the trajectory.
             The default is 0.0.
         nav : int, optional
-            Total number of time steps (frames) to consider.
-            The default is None.
+            Number of frames to consider for the time average.
+            The default is 10 if None, i.e., all 
```

Comment:
remove "if None, i.e., all available time steps are used." This option is only available for a few classes.
We want users to have a consistent experience. to use all time-steps they should give "traj.nframes" or np.inf if they are sketchy... ;)

Reply:
Done.